ansible/roles/edge/templates/bootstrap.sh

```diff
@@ -10,49 +10,70 @@ main() {
 
 {% for item in hostvars %}
 {% if (hostvars[item].tags.Role == "fullnode" or hostvars[item].tags.Role == "validator") %}
-    polygon-edge polybft-secrets init --data-dir {{ hostvars[item].tags["Name"] }} \
-    --chain-id {{ chain_id }} \
-    --json \
-    --insecure > {{ hostvars[item].tags["Name"] }}.json
+    polygon-edge polybft-secrets --data-dir {{ hostvars[item].tags["Name"] }} --json --insecure > {{ hostvars[item].tags["Name"] }}.json
 {% endif %}
 {% endfor %}
-
-    apt update
-    curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
-    apt-get install -y nodejs
-
-    pushd /opt/polygon-edge/
-    make compile-core-contracts
-    cp -r /opt/polygon-edge/core-contracts /var/lib/bootstrap/core-contracts/
-    popd
-
     polygon-edge genesis \
```

Comment:
Should this also create a test native token with `--native-token-config`?

Reply:
It definitely sounds like we should do that, but may defer to a different PR.